cms/djangoapps/contentstore/management/commands/tests/test_export_all_courses.py

```diff
@@ -20,8 +20,12 @@ def setUp(self):
         """ Common setup. """
         self.store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)
         self.temp_dir = mkdtemp()
-        self.first_course = CourseFactory.create(org="test", course="course1", display_name="run1")
-        self.second_course = CourseFactory.create(org="test", course="course2", display_name="run2")
+        self.first_course = CourseFactory.create(
```

Comment:
@cpennington Further down in the code (line 41), an operation is done on the store that doesn't work on mixed modulestore. We need to store the mongo modulestore as self.store.